lstchain/irf/hdu_table.py

```diff
@@ -0,0 +1,393 @@
+import numpy as np
+import logging
+
+import astropy.units as u
+from astropy.table import Table, QTable
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, EarthLocation, AltAz
+from astropy.time import Time
+
+from lstchain.__init__ import __version__
+
+__all__ = ["create_obs_hdu_index", "create_event_list"]
+
+log = logging.getLogger(__name__)
+
+DEFAULT_HEADER = fits.Header()
+DEFAULT_HEADER["CREATOR"] = f"lstchain v{__version__}"
+DEFAULT_HEADER["HDUDOC"] = "https://github.com/open-gamma-ray-astro/gamma-astro-data-formats"
+DEFAULT_HEADER["HDUVERS"] = "0.2"
+DEFAULT_HEADER["HDUCLASS"] = "GADF"
+DEFAULT_HEADER["ORIGIN"] = "CTA"
+DEFAULT_HEADER["TELESCOP"] = "CTA"
```

Comment:
`CTA-N-LST-1` for now? I think the TELESCOP should identify the subarray used if I remember discussions about this correctly

Reply:
Ok, I will amend it.

Reply:
Ah, yes, that makes more sense. Confused TELESCOP and INSTRUME. So maybe even CTA for TELESCOP and INSTRUME is CTA-N-LST-1 ? not sure, both should be fine.

Reply:
Yes, but I think it makes more sense to add the North/South information to TELESCOP and for INSTRUME we can keep just LST-1 and later it can include other MSTs and SSTs as well.
Either way, as we do not have a standard yet, we will have to change this in the future.